packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,114 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const PROJECT_NAME = getPaths().base.match(/[^/|\\]+$/)[0]
+
+const RENDER_YAML = (database) => {
+  return `services:
+- type: web
+  name: ${PROJECT_NAME}-web
+  env: static
+  buildCommand: yarn rw deploy render web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: NODE_VERSION
+    value: 14
+  routes:
+  - type: rewrite
+    source: /.redwood/functions/*
+    destination: replace_me_with_api_url/*
```

Comment:
TO DO (yes?)

Reply:
I had no idea this was possible in render, very awesome!

Reply:
The `replace_me_with_api_url/*` is currently a filler string that the user will need to update in our dashboard after deployment.

Reply:
Ah, got it. I'll try this full process out soon.

This one's just begging for me, as a new user, to figure out what I’m supposed to add there. Thoughts about:
- inlining a comment? # to update with a value from your Render dashboard (etc...)
- or making this an explicit Next step in the success message after `rw setup deploy render` is run: 1) Do this 2) and then go do this!

Reply:
✅